### ISSUE TITLE
Parsing fails on JSX + Flow file

### ISSUE THREAD

**Issue by bzz** [bblfsh/javascript-driver#59]
## Steps to reproduce
```
docker run --rm -it -p 9432:9432 bblfsh/javascript-driver:v2.6.0
wget https://raw.githubusercontent.com/kiwicom/nitrolib/91c1b4d26d7690690529ba0001f24985b161f2ac/src/components/LocationPicker/index.jsx
bblfsh-cli -l javascript index.jsx
```

## Expected
UASTv2 tree printed (with some nodes may be not annotated)

## Actual 

```
couldn't parse index.jsx: transform failed: check: key "specifiers": field was not used: leadingComments
```

**Comment by juanjux** [comment]
This is because the SDK was updated to 2.14.x but the annotations were not.

**Comment by creachadair** [comment]
> This is because the SDK was updated to 2.14.x but the annotations were not.

Does that mean we need to change something in the driver, or that we need only fix a version skew?

**Comment by juanjux** [comment]
@creachadair the driver's annotations must be updated if the SDK dependency is updated to SDK 2.14, the new version adds a check that will detect silently dropped fields in the semantic annotations so a previously working build would fail after the update. So the bug was there, just not detected.

**Comment by bzz** [comment]
@juanjux I do not think so - I think this is a real bug with current (before SDK update) annotations that happens on the `:latest` JS driver.

Changes related to update of the SDK are in #57 - there CI fails after SDK update but due to lack of new annotations (WIP on my side).

**Comment by juanjux** [comment]
True. The message is almost the same so I got that wrong.

**Comment by bzz** [comment]
Yes, that is very tricky - basically this seems like the same validation, but only in runtime.

Has been addressed in #60 and now I'm moving to doing the same for js as you already did for cpp/ruby/etc 😨
